classes/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
```

Comment:
- [ ] [Optional] It's considered a best practice if you add these setters and getters values before the `initialize` method.

Reply:
Implemented 👍

Reply:
@sja-thedude Thanks :bow: